.circleci/config.yml

```diff
@@ -76,6 +79,10 @@ jobs:
           command: |
             bundle check --path=/home/circleci/project/vendor/bundle || bundle install --deployment --path=/home/circleci/project/vendor/bundle --jobs=4 --retry=3 --without ''
 
+      - run:
+          name: set yarn version
+          command: sudo apt-get install yarn=1.6.0-1
```

Comment:
Can we set this up in our docker image instead of installing it every time?

Reply:
Good point.  Looks like we'd change that here:

https://github.com/code-dot-org/code-dot-org/blob/6f36db9d748931de7e5a53d7b0b5ceb56a28eefa/.circleci/Dockerfile#L23-L25

But I potentially need @wjordan to push the updated image to Docker hub?

Reply:
Discussed in person.  I'm going to go ahead and merge this with the install in the Circle config (takes about 4s) and expect follow-up work to move our docker image someplace with shared ownership, at which time we can move this up to the image.

Reply:
Did this ever happen?

> follow-up work to move our docker image someplace with shared ownership

Reply:
I suspect it did - the yarn version is set in the dockerfile here:

https://github.com/code-dot-org/code-dot-org/blob/09b4a135a281069a3375b95ccad1d495cc13f30c/docker/dockerfiles/Dockerfile#L32

@uponthesun may know more about where our image lives / how to update it.